### PR TITLE
레지스터 페이지 작업, 2차(StickBottomButton)

### PR DESCRIPTION
**Pull request by chepchep2**
- 레지스터 페이지 작업중입니다.
- 2차로 StickyBottomButton 작업중입니다.

### CODE REVIEW

**Comment by chepchep2** [comment]
* **#17** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/17?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#16** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/16?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#15** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/15?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#14** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/14?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#13** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/13?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#12** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/12?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#11** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/11?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#10** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/10?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#9** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/9?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#8** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/8?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#7** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/7?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#6** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/6?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#5** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/5?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/4?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#3** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/3?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/2?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `feature/introPage`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @chepchep2 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: lib/common/sticky_bottom_button.dart [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import 'package:flutter/material.dart';
+
+class StickyBottomButton extends StatefulWidget {
```
Comment (flowkater) [inline lib/common/sticky_bottom_button.dart:3]: State 가 없는데 굳이 Stateful 로 만들 필요는 없을 것 같습니다.
왠만하면 공용으로 사용하는 Widget 은 Stateless 가 되면 좋을 것 같아요.

**Review by flowkater — CHANGES_REQUESTED** [review]
코드 참고 부탁드립니다.

File: lib/common/sticky_bottom_button.dart (hunk 1 above)
Comment (chepchep2) [reply lib/common/sticky_bottom_button.dart:3]: 참고하겠습니다

**Comment by flowkater** [comment]
### Merge activity

* **Mar 7, 8:27 AM EST**: @flowkater started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/4).
* **Mar 7, 8:33 AM EST**: [Graphite](https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/4) rebased this pull request as part of a merge.
* **Mar 7, 8:34 AM EST**: @flowkater merged this pull request with [Graphite](https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/4).

**Review by flowkater — APPROVED** [review]

**Pull request merged**